vividus/src/test/java/org/vividus/transformer/InnerJoinTableTransformerTests.java

```diff
@@ -0,0 +1,151 @@
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.transformer;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.doReturn;
+
+import java.util.List;
+import java.util.Properties;
+
+import org.apache.commons.lang3.StringUtils;
+import org.jbehave.core.configuration.Keywords;
+import org.jbehave.core.model.ExamplesTable;
+import org.jbehave.core.model.ExamplesTable.TableProperties;
+import org.jbehave.core.steps.ParameterConverters;
+import org.jbehave.core.steps.ParameterConverters.FluentEnumConverter;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Spy;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class InnerJoinTableTransformerTests
+{
+    private static final String JOINT_COLUMN = "joinID";
+    private static final String OTHER_JOINT_COLUMN = "joinID1";
+    private static final String RIGHT_COLUMN_PARAMETER_NAME = "rightTableJoinColumn";
+    private static final String LEFT_COLUMN_PARAMETER_NAME = "leftTableJoinColumn";
+    private static final ExamplesTable FIRST_TABLE = new ExamplesTable(
+            "|column1|joinID|column2|column3|\n"
+                    + "|row11  |1     |row21  |row31  |\n"
+                    + "|row12  |2     |row22  |row32  |\n"
+                    + "|row13  |3     |row23  |row33  |\n"
+                    + "|row133 |3     |row233 |row333 |\n"
+                    + "|row14  |4     |row24  |row34  |\n"
+            );
+
+    private static final ExamplesTable SECOND_TABLE = new ExamplesTable(
+            "|joinID|column4|column5|\n"
+                    + "|5     |row45  |row51  |\n"
+                    + "|3     |row43  |row53  |\n"
+                    + "|1     |row41  |row51  |\n"
+                    + "|3     |row433 |row533 |\n"
+            );
+
+    private final Keywords keywords = new Keywords();
+    private final ParameterConverters parameterConverters = new ParameterConverters()
+            .addConverters(new FluentEnumConverter());
+
+    @Spy private InnerJoinTableTransformer transformer;
+
+    @Test
+    void shouldTransform()
+    {
+        var tableProperties = createProperties(JOINT_COLUMN, JOINT_COLUMN, StringUtils.EMPTY);
+        doReturn(List.of(FIRST_TABLE, SECOND_TABLE)).when(transformer).loadTables(StringUtils.EMPTY,
+                tableProperties);
+        String tableAsString = transformer.transform(StringUtils.EMPTY, null, tableProperties);
```

Comment:
either use `var` everywhere it's applicable or nowhere

Reply:
done